examples/smallvil/grabs/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod move_grab;
+pub use move_grab::MoveSurfaceGrab;
```

Comment:
Could we flatten the sub module into the grabs?

Probably would be easier to read otherwise

Reply:
Done